### ISSUE TITLE
Implementation of AQA

### ISSUE THREAD

**Issue by artofscience** [artofscience/SAOR#123]
I would suggest to "try" to implement the AQA (Li, Khandewal 2021), to test how "easy" it is to implement advanced schemes using the building blocks we have. This should come with appropriate tests (iterate values available from the paper).

**Comment by dirkmunro89** [comment]
AQA?

**Comment by artofscience** [comment]
It is the type of approximation schemes Giannis looked at during his thesis:

Li, L., & Khandelwal, K. (2015). An adaptive quadratic approximation for structural and topology optimization. Computers and Structures, 151, 130–147. https://doi.org/10.1016/j.compstruc.2015.01.013

**Comment by dirkmunro89** [comment]
Its quadratic QP? please send me the paper....

**Comment by dirkmunro89** [comment]
Yes ok; but this is essentially the same 'issue' / the combined 'issue' of approximated approximations and the trust-region filter / 'conditional acceptance of iterates'. There is equivalences between these things.
